Content.Server/Silicons/Borgs/BorgSystem.cs

```diff
@@ -281,9 +281,12 @@ private void UpdateBatteryAlert(Entity<BorgChassisComponent> ent, PowerCellSlotC
     /// </summary>
     public void BorgActivate(EntityUid uid, BorgChassisComponent component)
     {
+
         Popup.PopupEntity(Loc.GetString("borg-mind-added", ("name", Identity.Name(uid, EntityManager))), uid);
         Toggle.TryActivate(uid);
         _appearance.SetData(uid, BorgVisuals.HasPlayer, true);
+        if (_mind.TryGetMind(uid, out _, out var mind))
+            _powerCell.SetDrawEnabled(uid, true);
     }
 
     /// <summary>
```

Comment:
This looks hacky, why doesn't deactivate set it, why isn't this set on the yaml instead, etc.

Reply:
First of because i didnt think of setting it in the yaml and i am not sure i can but i can deffinatly check that, and I only added this because normaly setting it happens every time there is a `MobStateChangedEvent` that eqauls `MobState.Alive` in the function `OnMobStateChanged`, but this doesnt recognize the start of the round as such so fails to enable it correctly. But it does change it after something like the borg being knocked down etc. I only added this to ensure it being enabled on round start. I could rewrite it so it ONLY gets changed in enable and disable but i am unsire what other consiqunzes that might have.

Reply:
I brifly took a look around and i know my inabbiltiy isnt an excuse so id'love to learn how to set it in the yaml if possible.